remoteit/desktop
# [DESK-296][DESK-375] Graphql API integration

### Description


Change the device list query from the old REST API to use the new GraphQL API.

### Checklist




### Test plan



1. Sign in
2. Go to the device list
3. Search, sort, filter, load more. 
4. You should see it work for all size device lists even with 1000 devices

### Ticket



<https://remoteit.atlassian.net/browse/DESK-296>
<https://remoteit.atlassian.net/browse/DESK-375>
